tests/basic/bug71273.phpt

```diff
@@ -4,7 +4,7 @@ Bug #71273 A wrong ext directory setup in php.ini leads to crash
 <?php
     /* NOTE this file is required to be encoded in iso-8859-1 */
 
-    $cmd = getenv('TEST_PHP_EXECUTABLE') . " -n -d html_errors=on -d extension_dir=a/�/w -d extension=php_kartoffelbrei.dll -v 2>&1";
+    $cmd = escapeshellarg(getenv('TEST_PHP_EXECUTABLE')) . " -n -d html_errors=on -d extension_dir=a/�/w -d extension=php_kartoffelbrei.dll -v 2>&1";
```

Comment:
the encoding/char must be unchanged

Reply:
Thanks, this indeed slipped through, I'll fix it.